funccount.py

```diff
@@ -0,0 +1,30 @@
+import os
+import signal
+import subprocess as sp
+import time
+
+class FUNCCOUNT:
+    def __init__(self, funcs, sleep=0):
+        self.funcs = funcs
+        self.sleep = sleep
+
+    def start(self):
+        cmd = f"sleep {self.sleep} && funccount {self.funcs}"
+
+        self.run = sp.Popen(cmd, shell=True, stdout=sp.PIPE, text=True,
+                            start_new_session=True)
+
+    def stop(self):
+        pgid = os.getpgid(self.run.pid)
+        os.killpg(pgid, signal.SIGINT)
+
+        while self.run.poll() is None:
+            time.sleep(0.1)
+
+        res = {}
+        for line in self.run.stdout.readlines():
```

Comment:
Really seems like we should be using Popen.communicate() here....

If funccount outputs more data than what is buffered this will hang in the while loop above (which is a poor substitute for Popen.wait())....